linux_os/guide/system/permissions/files/permissions_within_important_dirs/dir_permissions_library_dirs/rule.yml

```diff
@@ -0,0 +1,57 @@
+documentation_complete: true
+
+title: |-
+{{% if product in ["sle15"] %}}
+    The SUSE operating system library directories must have mode 0755 or less permissive.
+{{% else %}}
+    Verify that Shared Library Directories Have Restrictive Permissions
+{{% endif %}}
+
+description: |-
+    System-wide shared library directories, which contain are linked to executables
+    during process load time or run time, are stored in the following directories
+    by default:
+    <pre>/lib
+    /lib64
+    /usr/lib
+    /usr/lib64
+    </pre>
+    Kernel modules, which can be added to the kernel during runtime, are
+    stored in <tt>/lib/modules</tt>. All sub-directories in these directories
+    should not be group-writable or world-writable. If any file in these
+    directories is found to be group-writable or world-writable, correct
+    its permission with the following command:
+    <pre>$ sudo chmod go-w <i>DIR</i></pre>
+
+rationale: |-
+    If the SUSE operating system were to allow any user to make changes to software libraries,
```

Comment:
Don't think this should be SUSE specific.

Reply:
thanks - removed that in 4e4ea53